Cargo.toml

```diff
@@ -17,6 +17,7 @@ mocks = ["mockall"]
 os_keyring = ["keyring"]
 
 [dependencies]
+tendermint-rpc = { version = "0.28", features = ["websocket-client"] }
```

Comment:
Lets use the version cosmrs v0.10.0 is currently using to simplify things here (tendermint-rpc `0.26`)